### DISCUSSION TITLE
ROCm 5.6.0 Release

### DISCUSSION THREAD

**Discussion by saadrahim** [ROCm/ROCm#2300] [Announcements]
# Release Highlights

ROCm 5.6 consists of several AI software ecosystem improvements to our fast-growing user base. A few examples include:

- New documentation portal at https://rocm.docs.amd.com with highlights on our accompanying [blog](https://community.amd.com/t5/rocm/rocm-docs-beginning-a-new-journey/ba-p/611382)
- Ongoing software enhancements for LLMs, ensuring full compliance with the HuggingFace unit test suite
- OpenAI Triton, CuPy, HIP Graph support, and many other library performance enhancements
- Improved ROCm deployment and development tools, including CPU-GPU (rocGDB) debugger, profiler, and docker containers
- New pseudorandom generators are available in rocRAND.  Added support for half-precision transforms in hipFFT/rocFFT.  Added LU refactorization and linear system solver for sparse matrices in rocSOLVER.  

Please see the complete [release notes](https://rocm.docs.amd.com/en/docs-5.6.0/release.html).

<hr /><em>This discussion was created from the release <a href='https://github.com/RadeonOpenCompute/ROCm/releases/tag/rocm-5.6.0'>ROCm 5.6.0 Release</a>.</em>

**Comment by Kila2** [comment]
Does it support windows?

**Reply by countradooku** [reply]
Or RdNA 3 support

**Reply by saadrahim** [reply]
> Or RdNA 3 support

Your comment was deleted due to the use of abusive language. Please keep the conversation civil.

**Reply by johnnynunez** [reply]
Its official https://twitter.com/amdradeon/status/1674438636488589313?s=46

**Reply by countradooku** [reply]
damn....so much tiume to wait. :(. and yet nothing about , windows release

**Reply by Lumpsack** [reply]
I do wish they'd hurry up with the Windows part, and because I'm greedy RdNA 3 too :)

**Reply by wsippel** [reply]
RDNA3 might not be _officially_ supported until some time later this year, but it works fine. Already worked fine on ROCm 5.5, actually. Though performance probably isn't where it should be, and stuff like Triton or Torch Inductor didn't work in ROCm 5.5, and I've not tested 5.6 yet.

**Reply by countradooku** [reply]
yeah and they dont use that AI cores yet I bet. That would give a huge performance boost too

**Reply by wsippel** [reply]
You can access the AI cores via Composable Kernel or rocWMMA. I believe AITemplate uses them.

**Reply by countradooku** [reply]
The problem is i can;t use ubuntu 22.04.2 which is required because of the kernel. It boots me in a black screen with an GPU init error. Any solutions for that? I could install rocm on newer distros but i cant install the dkms driver which is also required for DL frameworks...

**Reply by wsippel** [reply]
I don't think you need the DKMS drivers if you run a sufficiently recent kernel. I don't have them installed and ROCm works fine. I'm on Arch though, running kernel 6.3.9.

**Reply by countradooku** [reply]
WHat frameworks do you use?

**Reply by countradooku** [reply]
i mean tensorflow torch etc

**Reply by wsippel** [reply]
Torch.

**Reply by countradooku** [reply]
ok thanks

**Reply by countradooku** [reply]
And you use rocwmma?

**Reply by johnnynunez** [reply]
https://github.com/RadeonOpenCompute/ROCm/discussions/2347

**Comment by lhl** [comment]
The release notes is linking to 5.5.1 for some reason. Here's the correct link: https://rocm.docs.amd.com/en/docs-5.6.0/release.html

**Reply by saadrahim** [reply]
Thanks for noticing. I have fixed the link.

**Comment by YellowRoseCx** [comment]
thank you for all the hard work you and the rest of the AMD team has put into improving ROCm!


The only issue I saw with the install instructions was with this codeblock:
`# amdgpu repository for jammy
echo 'deb [arch=amd64 signed-by=/etc/apt/keyrings/rocm.gpg] https://repo.radeon.com/amdgpu/5.6/ubuntu jammy main' \
    | sudo tee /etc/apt/sources.list.d/amdgpu.list
sudo apt update`

I'm using Linux Mint 21.1 Cinnamon and my rocm key was not in that location. my rocm key was in /etc/apt/trusted.gpg.d/rocm-keyring.gpg so I had to install it with this:

`deb [arch=amd64 signed-by=/etc/apt/trusted.gpg.d/rocm-keyring.gpg] https://repo.radeon.com/amdgpu/5.6/ubuntu jammy main`
